extensions/adobe/experience/journeyOrchestration/stepEvents/journeyClass.json

```diff
@@ -0,0 +1,29 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journey",
```

Comment:
Get the $id to refer to the right container under the experience folder, in this case it should be the campaign.

Reply:
As Journey Orchestration is a separate product so use this new path.